### PR TITLE
winget: Update to version 1.3.2091 (Stable)

### PR DESCRIPTION
**Pull request by delassiops**
- Update Url to Stable Release
- use checkver: "github"

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by delassiops** [comment]
already verified

File: bucket/winget.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-    "version": "1.4.2161",
-    "description": "Windows Package Manager CLI (aka winget)",
+    "version": "1.3.2091",
+    "description": "Windows Package Manager Client (aka winget)",
```
[the remark this answers is not in the document]
Comment (rashil2000) [reply bucket/winget.json:3]: Change this back to original description

[the remark this answers is not in the document]
Comment (niheaven) [reply bucket/winget.json:3]: image

It looks like the repo has two short description

**Comment by rasa** [comment]
Is the thought that the preview version be available in the Versions bucket?

**Comment by niheaven** [comment]
> Is the thought that the preview version be available in the Versions bucket?

Added in https://github.com/ScoopInstaller/Versions/commit/3cf9ee4c8dbc2743bd660df42320e25db6e38a1e

**Pull request merged**